tests/pytests/unit/test_crypt.py

```diff
@@ -4,10 +4,100 @@
 
 Unit tests for salt's crypt module
 """
+
+import uuid
+
 import pytest
 import salt.crypt
+import salt.master
 import salt.utils.files
 
+PRIV_KEY = """
```

Comment:
Should these be fixtures instead of globals?

Reply:
Considering they are just static strings with no setup and teardown. I don't think it matters. Maybe a purist would disagree.

Reply:
Comment based on previous reviews where we seem to want to avoid using global variables in tests.  Not a blocker, just something I noticed when reviewing.

Reply:
Mutable objects as global variable is the **real** problem, strings are ok, if we must, and given that these are long strings, which we don't want to maintain on separate files, these are ok.